scss/_o-step-bar.scss

```diff
@@ -54,6 +54,8 @@
   max-width: 1.25ch;
   margin: auto;
   overflow: hidden;
+  font-size: $font-size-sm;
+  line-height: $o-stepbar-line-height;
```

Comment:
I'd use here `line-height: $line-height-sm;` here.

Isn't there a bug with `$line-height-sm`? When I apply it, the set value is `calc(divide(16, 14))` and the browser can't use it. Small buttons seems broken as well.

Reply:
Yes it looks like a bug, I guess that it comes from lines 280 and 281 in the `_variables.scss`, the calc function is too much there. At least it works well without.

Reply:
Issue created: https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1066

Reply:
This issue is now closed. Resolving this conversation and rebasing this PR.